Cython/Compiler/Nodes.py

```diff
@@ -5538,6 +5547,7 @@ class PropertyNode(StatNode):
     #  body   StatListNode
 
     child_attrs = ["body"]
+    created_from_decorator = 0  # just used in a deprecation warning
```

Comment:
```suggestion
    created_from_decorator = False  # just used in a deprecation warning
```
Regarding the comment, I'm not sure if this definition should know about the usages. They might change at some point, and the comment is unlikely to get updated then. I'd leave this to the commit comment.